west.yml

```diff
@@ -0,0 +1,44 @@
+# The west manifest file for Nordic Connect SDK, NCS.
+#
+# The per-installation west configuration file, .west/config, sets the
+# path to the project containing this file in the [manifest] section's
+# "path" variable.
+#
+# You are free to create your own manifest files and put them in any
+# repository you want, to create your own custom NCS installation.
+# For example, you could create a manifest file in your own out of
+# tree application directory, which would pull this nrf repository
+# in as an ordinary project.
+#
+# You can pass your manifest repositories to west init when creating a
+# new NCS installation. See the west documentation for more
+# information.
+
+west:
+  url: https://github.com/zephyrproject-rtos/west
+  revision: master
+
+manifest:
+  defaults:
+    remote: upstream
```

Comment:
I would name this "ncs" and add a comment that this is the "nRF Connect SDK"

Reply:
Is this the name of the remote on `git`? (disclaimer: mine's called `upstream` :)

Reply:
> I would name this "ncs" and add a comment that this is the "nRF Connect SDK"

agree

> Is this the name of the remote on `git`

no, it's not the name of the remote in `git`
Only internal name used in `west`